### PR TITLE
Fix `Jailbird::WearState`

### PR DESCRIPTION
**Pull request by louis1706**
Also Add Jailbird Propperty inside SwingingEvent

### CODE REVIEW

File: EXILED/Exiled.API/Features/Items/Jailbird.cs [hunk 1]
```diff
@@ -114,12 +117,44 @@ public JailbirdWearState WearState
...
         }
 
+        /// <summary>
+        /// Calculates the damage corresponding to a given <see cref="JailbirdWearState"/>.
+        /// </summary>
+        /// <param name="wearState">The wear state to calculate damage for.</param>
+        /// <returns>The amount of damage associated with the specified wear state.</returns>
+        public float GetDamage(JailbirdWearState wearState)
+        {
+            foreach (Keyframe keyframe in Base._deterioration._chargesToWearState.keys)
```
Comment (Misfiy) [inline EXILED/Exiled.API/Features/Items/Jailbird.cs:133]: i believe this is wrong

**Review by Misfiy — CHANGES_REQUESTED** [review]

File: EXILED/Exiled.API/Features/Items/Jailbird.cs (hunk 1 above)
Comment (louis1706) [reply EXILED/Exiled.API/Features/Items/Jailbird.cs]: yup thanks

File: EXILED/Exiled.API/Features/Items/Jailbird.cs [hunk 2]
```diff
@@ -114,12 +117,44 @@ public JailbirdWearState WearState
...
+
+            throw new Exception("Wear state not found in charges to wear state mapping.");
+        }
+
+        /// <summary>
+        /// Gets the charge needed to reach a specific <see cref="JailbirdWearState"/>.
+        /// </summary>
+        /// <param name="wearState">The desired wear state to calculate the charge for.</param>
+        /// <returns>The charge value required to achieve the specified wear state.</returns>
+        public int GetCharge(JailbirdWearState wearState)
```
Comment (ITeMbI4) [inline EXILED/Exiled.API/Features/Items/Jailbird.cs:147]: It will not work

Comment (louis1706) [reply EXILED/Exiled.API/Features/Items/Jailbird.cs:147]: > It will not work

Why that how nw does it ?

Comment (ITeMbI4) [reply EXILED/Exiled.API/Features/Items/Jailbird.cs:147]: https://github.com/Exiled-Team/EXILED/pull/2736

Comment (louis1706) [reply EXILED/Exiled.API/Features/Items/Jailbird.cs]: Resolved

**Comment by Jesus-QC** [comment]
Why are we deleting those files, please explain all the changes in the pr body

**Review by Misfiy — APPROVED** [review]

**Review by Jesus-QC — APPROVED** [review]

**Review by Misfiy — APPROVED** [review]

**Pull request merged**